src/main/java/org/trigon/utilities/mirrorable/MirrorablePose2d.java

```diff
@@ -4,20 +4,42 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.geometry.Translation2d;
 
-
+/**
+ * A class that represents a {@link Pose2d} that can be mirrored. Reversing its position and orientation.
```

Comment:
The second sentence shouldn't be on its own. Also, this doesn't explain how it is mirrored. How would someone reading this know if it mirrors it for a different alliance or just for something different.